hadoop-hdds/interface-server/src/main/proto/SCMRatisProtocol.proto

```diff
@@ -23,6 +23,7 @@ option java_generate_equals_and_hash = true;
 enum RequestType {
     PIPELINE = 1;
     CONTAINER = 2;
+    DELETE_BLOCK = 3;
 }
```

Comment:
should we make this request type to be just BLOCK rather than DELETE_BLOCK ?
For now, for blocks we only need to handle delete block info, but in future if we have to extend this for some block specific attributes, it would be more simple.

For understanding as well, we can think of requests opearting at 3 levels, pipeline, container and blocks.